source/ur/ur.hpp

```diff
@@ -302,7 +302,8 @@ class UrReturnHelper {
 
   // Array return value where element type is differrent from T
   template <class RetType, class T>
-  ur_result_t operator()(const T *t, size_t s) {
+  typename std::enable_if<!std::is_same<RetType, T>::value, ur_result_t>::type
```

Comment:
We're a C++17 codebase so this could use `std::is_same_v` instead.

```suggestion
  typename std::enable_if<!std::is_same_v<RetType, T>, ur_result_t>::type
```

Reply:
And likewise for `std::enable_if_t`, done, thanks.